### ISSUE TITLE
Task ordering

### ISSUE THREAD

**Issue by Ekleog** [Ekleog/todiff#20]
Since #17, the postponed are no longer sorted at the top of the “Changed tasks” section (and I'd assume recurred tasks are no longer at the top of the “Completed tasks” section, but don't have a test case to check right now).

Example (with `--similarity=50`):
```
# FROM

foo due:2018-07-04
bar due:2018-07-04

# TO

fool due:2018-07-04
bar due:2018-07-05
```

Observing the order of the changes before and after #17.

cc @Nadrieril
